packages/gestalt/src/SideNavigationItem.js

```diff
@@ -0,0 +1,155 @@
+// @flow strict
+import { useState, type Node } from 'react';
+import classnames from 'classnames';
+import styles from './SideNavigation.css';
+import TapArea from './TapArea.js';
+import Badge from './Badge.js';
+import Icon from './Icon.js';
+import Flex from './Flex.js';
+import Text from './Text.js';
+import Box from './Box.js';
+import icons from './icons/index.js';
+
+type Props = {|
+  /**
+   * When set to 'page' or 'section', it displays the item in "active" state. See the [Accessibility](https://gestalt.pinterest.systems/SideNavigation#Accessibility) guidelines to learn more.
+   */
+  active?: 'page' | 'section',
+  /**
+   * When supplied, will display a [Badge](https://gestalt.pinterest.systems/badge) next to the item's label. See the [Badges](https://gestalt.pinterest.systems/SideNavigation#Badge) variant to learn more.
+   */
+  badge?: {|
+    text: string,
+    type?: 'info' | 'error' | 'warning' | 'success' | 'neutral' | 'darkWash' | 'lightWash',
```

Comment:
I think we should remove light wash and dark wash, they're only intended for use over images, and that shouldn't happen here

Reply:
DONE